### PR TITLE
fix collections.abc deprecation

### PR DESCRIPTION
**Pull request by akaszynski**
Fix Python 3.10 bug due to deprication of importing from `collections.abc`.  See:
https://programmerah.com/solved-python-using-or-importing-the-abcs-from-collections-instead-of-from-collections-abc-is-deprecate-32596/

---

I'll be putting #2044 into draft as it's too much to "officially" support Python 3.10 via CI/CD until Kitware can generate a wheel for Python 3.10. I'm hoping based on their latest press releases that it will happen this Spring with the release of 9.2.

### CODE REVIEW

**Comment by banesullivan** [comment]
Ugh. These GitHub job `if:` conditions are killing me. This should have been autolabeled as a `bug-fix`:

https://github.com/pyvista/pyvista/blob/123b8833e35ddd811bdd84b49b24403a93b72203/.github/workflows/label.yml#L36

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 CODE_DIRS ?= doc examples examples_flask pyvista tests
 # Files in top level directory
 CODE_FILES ?= *.py *.rst *.md
-CODESPELL_SKIP ?= "*.pyc,*.txt,*.gif,*.png,*.jpg,*.ply,*.vtk,*.vti,*.vtu,*.js,*.html,*.doctree,*.ttf,*.woff,*.woff2,*.eot,*.mp4,*.inv,*.pickle,*.ipynb,flycheck*,./.git/*,./.hypothesis/*,*.yml,./doc/_build/*,./doc/images/*,./dist/*,*~,.hypothesis*,./doc/examples/*,*.mypy_cache/*,*cover,./tests/tinypages/_build/*,*/_autosummary/*"
+CODESPELL_SKIP ?= "*.pyc,*.txt,*.gif,*.png,*.jpg,*.ply,*.vtk,*.vti,*.vtu,*.js,*.html,*.doctree,*.ttf,*.woff,*.woff2,*.eot,*.mp4,*.inv,*.pickle,*.ipynb,flycheck*,./.git/*,./.hypothesis/*,*.yml,./doc/_build/_static/*,./doc/images/*,./dist/*,*~,.hypothesis*,./doc/examples/*,*.mypy_cache/*,*cover,./tests/tinypages/_build/*,*/_autosummary/*"
```
Comment (adeak) [inline Makefile:7]: Why is this necessary? Aren't all things in `_build/` built from checked source?

**Review by banesullivan — APPROVED** [review]

File: pyvista/core/pointset.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 """Sub-classes and wrappers for vtk.vtkPointSet."""
-import collections
+from collections.abc import Iterable
```
Comment (adeak) [inline pyvista/core/pointset.py:2]: Ah, nice, I didn't know they finally pulled the plug on this one. This was deprecated in 3.3! https://docs.python.org/3.9/library/collections.html

File: Makefile (hunk 1 above)
Comment (akaszynski) [reply Makefile:7]: Not quite. There's lots in `_build` that's autogenerated:
```
Running codespell
doc/_build/html/_static/pygments.css:40: nd ==> and, 2nd
doc/_build/html/_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.svg:147: adn ==> and
```

Comment (adeak) [reply Makefile:7]: @akaszynski according to the diff the original had
```
./doc/_build/*
```
while the new version has 
```
./doc/_build/_static/*
```
This should mean that originally we had "anything in `_build`, and now we have "anything in `_build/static`.

Since this is `CODESPELL_SKIP` these translate to being

> Skip everything in `_build/`

and

> Skip everything in `_build/static`

respectively. Reversing the logic for clarity's sake these are

> Check everything outside `_build`

and

> Check everything outside `_build/static`

respectively.

In other words, the change seems to _newly include_ things in codespell that are in `_build/` but outside `_build/static`. My point was that if everything in `_build` is autogenerated then why do we have to be stricter with the codespell skip?

Is it clearer now?
